rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2252,6 +2252,19 @@ def test_upgrade_db_41_to_42(user_data_dir, messages_aggregator):
         assert table_exists(cursor, 'zksynclite_transactions') is False
         assert table_exists(cursor, 'calendar') is False
         assert table_exists(cursor, 'calendar_reminders') is False
+
+        # history events that need to be deleted because their transaction are not in the database
+        expected_events_ids = [(17987,), (17988,), (17989,)]
+        assert cursor.execute(
```

Comment:
Shouldn't you also mark them as customized to emulate the real scenario this happens in?

Reply:
Yes, they are marked as customized